src/main/java/org/opentripplanner/apis/gtfs/datafetchers/QueryTypeImpl.java

```diff
@@ -890,6 +896,28 @@ public DataFetcher<Object> viewer() {
     return environment -> new Object();
   }
 
+  /**
+   * This matches station's feedScopedId to the given string.
+   */
+  private boolean isMatchRentalStationIds(VehicleRentalStation station, String feedScopedId) {
```

Comment:
I find this method name a little hard to parse. Can you improve it?

Reply:
I completely agree that the method names are awful but I wasn't quite sure if the method name should start with `is`. Instead of using the word `match` we could use `equal` but even then it's as messy. I'm open to suggestions.

Reply:
How about `matchesStationId` or `stationIdMatches` or `stationIdIsMatch`?